lib/active_record_shards-5-0.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'active_record_shards/schema_dumper_extension'
+
+
```

Comment:
Yeah, I added extra blank lines to these files, so they can easily be compared side by side, so you can see which hacks are enabled where.